app/views/entries/_form.html.erb

```diff
@@ -27,22 +27,26 @@
   </div>
   <div class="form-group">
     <div class="col-sm-6">
+      <label>Work Performed</label>
       <label for="entry_description" class="control-label sr-only">Description</label>
-      <textarea class="required form-control" name="entry[description]" id="entry_description" placeholder="Description"></textarea>
+      <textarea rows="10" class="required form-control" name="entry[description]" id="entry_description" placeholder="Description"></textarea>
       <%= error_span(@entry[:description]) %>
     </div>
     <div class="col-sm-6">
-      <label for="entry_parts_used" class="control-label sr-only">Parts used</label>
-      <textarea class="required form-control" name="entry[parts_used]" id="entry_parts_used" placeholder="Parts used"></textarea>
+      <label>Parts Used</label>
+      <label for="entry_parts_used" class="control-label sr-only">Part Numbers, Serial Numbers, Description, etc...</label>
```

Comment:
This one you actually went to the trouble of making fancy words for screen readers but didn't associate the label with the input.